CHANGELOG_API.md

```diff
@@ -0,0 +1,54 @@
+## API v0 alpha
+
+On 2023-03-02 endpoints were:
+
+- "All products" (`/api/all.json`) : Return a list of all products.
+- "Product" (`/api/{product}.json`) : Get EoL dates of all cycles of a given product.
+- "Cycle" (`/api/{product}/{cycle}.json`) : Details of a single release cycle of a given product.
+
+## API v1.0.0-b1
+
+### Summary
+
+API v1 is a major rework of the API v0 with a lot of breaking changes. Compared the API v0, API v1:
+
+- is generated using a Jekyll Generator (see https://jekyllrb.com/docs/plugins/generators/),
+- is versioned using the `api/v1` prefix, making it easier to implement non-backward-compatible
+  changes,
+- feels more "Restful".
+
+API v1 resolves : #394, #759, #905, #2062, #2066, #2078, #2160, #2331, #2431, #2595. It also reverts
+#2425 due to incompatibilities in redirect rules.
+
+The API v0 is still generated to give time to users to migrate to API v1.
```

Comment:
I guess it may depend on how quickly users migrate, but can we give a rough idea of how long v0 will still be available? That will help people plan migration.

Reply:
For the time being v0 does not require much maintenance. And I do not see it becoming a problem soon because we can emulate the old frontmatter format using `product-data-enricher.rb`. So maybe we could say 1 year from the API v1 release date ? Would that be enough ?

@endoflife-date/everyone, any opinion regarding this subject ?

Reply:
1 year sounds good. We could re-enable Netlify Analytics at the end of the year and evaluate if we need to keep is running further. If there's not much maintenance required, I don't mind keeping it even further.

Reply:
1 year is :ok_hand: 
And when I see all the data that will available, it will motivate to put effort on migrations arount it :100: